app/src/main/res/layout/home_fragment.xml

```diff
@@ -16,32 +16,6 @@
     android:background="@color/white"
     android:orientation="vertical">
 
-    <LinearLayout
-      android:layout_width="match_parent"
-      android:layout_height="wrap_content"
-      android:background="@color/white"
-      android:gravity="center"
-      android:orientation="vertical">
-
-      <Button
```

Comment:
If i am supposed to show exploration button please comment.

Reply:
I suggest you keep this button for now, otherwise developer's wont be able to run test cases in any file.

Reply:
Ok done

Reply:
+1